docs/quick_start/installing.rst

```diff
@@ -15,9 +15,9 @@
 Installing Fledge
 ==================
 
-Fledge is extremely lightweight and can run on inexpensive edge devices, sensors and actuator boards.  For the purposes of this manual, we assume that all services are running on a Raspberry Pi running the Raspbian operating system. Be sure your system has plenty of storage available for data readings.
+Fledge is extremely lightweight and can run on inexpensive edge devices, sensors and actuator boards.  For the purposes of this manual, we assume that all services are running on a Raspberry Pi running the Bullseye operating system. Be sure your system has plenty of storage available for data readings.
 
-If your system does not have Raspbian pre-installed, you can find instructions on downloading and installing it at https://www.raspberrypi.org/downloads/raspbian/.  After installing Raspbian, ensure you have the latest updates by executing the following commands on your Fledge server::
+If your system does not have Bullseye pre-installed, you can find instructions on downloading and installing it at https://www.raspberrypi.org/downloads/bullseye/.  After installing Bullseye, ensure you have the latest updates by executing the following commands on your Fledge server::
```

Comment:
May be we should use new URL here: `https://www.raspberrypi.com/software/operating-systems/`, though used one it redirects to the same